### PR TITLE
Remove X and tilda bodies

### PR DESCRIPTION
**Pull request by bethanyaconnor**

### CODE REVIEW

File: test/unit/qualityTest.test.js [hunk 1]
```diff
@@ -200,14 +200,14 @@ describe('Model quality test', () => {
   // });
 
   test('test eels with sharp teeth', async () => {
-    const trainSize = 300; // Need more fish to hit enough to train on
+    const trainSize = 500; // Need more fish to hit enough to train on
```
Comment (bethanyaconnor) [inline test/unit/qualityTest.test.js]: I had to increase this now that we only have one eel body...

Comment (uponthesun) [reply test/unit/qualityTest.test.js]: That worries me - I thought we agreed to make each distinct "kind" of part more equally represented. Seems like we should get rid of both eels or neither.

**Comment by bethanyaconnor** [comment]
Okay I'll update the test with a new test.

**Comment by bethanyaconnor** [comment]
Closing in favor of #293

**Pull request closed without merging**